docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/setup.md

```diff
@@ -105,6 +105,41 @@ Clone the EdgeX compose repository:
 git clone https://github.com/edgexfoundry/edgex-compose.git
 ```
 
+## Proxy Setup (Optional)
+
+!!! Note
+    The device used for deployment of device-onvif-service must be behind proxy/VPN.
```

Comment:
Do you mean to say that the user should only follow these steps if their device is behind a proxy/VPN? Also I am not sure if just being behind a VPN would require proxy setup. I think it would depend on if the VPN network required a proxy or not.

Reply:
Right, this process is only required if the end user's environment has a proxy. I will rephrase